caesarsCipher/script.js

```diff
@@ -0,0 +1,47 @@
+const form = document.getElementById('my-form');
+let userInput = document.getElementById('user-input');
+let result = document.getElementById('ciphered-word');
+const copyButton = document.getElementById('copy-btn');
+const copied = document.getElementById('copied');
+
+const TIMEOUT = 3000;
+
+function rot13(str) {
+    let arr = str.toUpperCase().split('');
+    let newArr = [];
+
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i].charCodeAt() < 65 || arr[i].charCodeAt() > 90) {
```

Comment:
Seems like 13, 65, 90 are config variables.
You can move them to constants.

Reply:
To make it even more interesting you can 
- add a new number input on UI, where the user can set the padding for the Cipher. 
- the value should be 13 by default.

Reply:
fixed